### PR TITLE
(Added) - Added danger and swiftlint to the project

### PR DESCRIPTION
**Pull request by pedro-nadolny**
## Kind of pr:

- [ ] BugFix 
- [ ] Feature
- [x] Improvement
- [ ] Refactor

### CODE REVIEW

**Comment by pedro-nadolny** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry. <br />
You can find it at <a href="https://github.com/mercadopago/px-ios/blob/develop/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR does not include tests, review carefully</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR, consider splitting into smaller ones.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">JIRA: <a href="https://mercadolibre.atlassian.net/browse/JIRA-123">JIRA-123</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger_pr">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by rgrajalesmeli — APPROVED** [review]
LG!

File: MercadoPagoSDK/MercadoPagoSDK/Core/Checkout/ViewModel/MercadoPagoCheckoutViewModel.swift [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import UIKit
 
-internal enum CheckoutStep: String {
+internal enum CheckoutStep: String  {
```
Comment (pedro-nadolny) [inline MercadoPagoSDK/MercadoPagoSDK/Core/Checkout/ViewModel/MercadoPagoCheckoutViewModel.swift:3]: <table data-meta="generated_by_danger_pr">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mercadopago/px-ios/blob/896d6540cb4a54ff23cb03976915d4293c13be96/MercadoPagoSDK/MercadoPagoSDK/Core/Checkout/ViewModel/MercadoPagoCheckoutViewModel.swift#L3"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>MercadoPagoCheckoutViewModel.swift:3</code></td>
    </tr>
  </tbody>
</table>

**Comment by pedro-nadolny** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Big PR, consider splitting into smaller ones.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR does not include tests, review carefully</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The <code>Podfile</code> was updated</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">JIRA: <a href="https://mercadolibre.atlassian.net/browse/JIRA-123">JIRA-123</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger_pr">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by pedro-nadolny** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Big PR, consider splitting into smaller ones.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR does not include tests, review carefully</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The <code>Podfile</code> was updated</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">JIRA: <a href="https://mercadolibre.atlassian.net/browse/JIRA-123">JIRA-123</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger_pr">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by pedro-nadolny** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR does not include tests, review carefully</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The <code>Podfile</code> was updated</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR, consider splitting into smaller ones.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">JIRA: <a href="https://mercadolibre.atlassian.net/browse/JIRA-123">JIRA-123</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger_pr">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by pedro-nadolny** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR does not include tests, review carefully</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The <code>Podfile</code> was updated</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR, consider splitting into smaller ones.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">JIRA: <a href="https://mercadolibre.atlassian.net/browse/JIRA-123">JIRA-123</a></td>
    </tr>
  </tbody>
</table>

## Current coverage for MercadoPagoSDKV4.framework is `2.89%`
✅ *No files affecting coverage found*

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_danger_pr">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by rgrajalesmeli — APPROVED** [review]
LG!

**Comment by pedro-nadolny** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR does not include tests, review carefully</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The <code>Podfile</code> was updated</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR, consider splitting into smaller ones.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">JIRA: <a href="https://mercadolibre.atlassian.net/browse/JIRA-123">JIRA-123</a></td>
    </tr>
  </tbody>
</table>

## Current coverage for MercadoPagoSDKV4.framework is `2.89%`
✅ *No files affecting coverage found*

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_danger_pr">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by pedro-nadolny** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR does not include tests, review carefully</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The <code>Podfile</code> was updated</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR, consider splitting into smaller ones.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">JIRA: <a href="https://mercadolibre.atlassian.net/browse/JIRA-123">JIRA-123</a></td>
    </tr>
  </tbody>
</table>

## Current coverage for MercadoPagoSDKV4.framework is `2.89%`
✅ *No files affecting coverage found*

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_danger_pr">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by pedro-nadolny** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry. <br />
You can find it at <a href="https://github.com/mercadopago/px-ios/blob/develop/CHANGELOG.md">CHANGELOG.md</a>. The changelog entry should be equal to the PR title in the correct section: Danger</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR does not include tests, review carefully</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The <code>Podfile</code> was updated</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR, consider splitting into smaller ones.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">JIRA: <a href="https://mercadolibre.atlassian.net/browse/JIRA-123">JIRA-123</a></td>
    </tr>
  </tbody>
</table>

## Current coverage for MercadoPagoSDKV4.framework is `2.89%`
✅ *No files affecting coverage found*

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_danger_pr">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by pedro-nadolny** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a <a href="https://github.com/mercadopago/px-ios/blob/develop/CHANGELOG.md">CHANGELOG.md</a> entry. The changelog entry should be equal to the PR title in the correct section: <strong>Added danger and swiftlint to the project</strong></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR does not include tests, review carefully</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The <code>Podfile</code> was updated</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR, consider splitting into smaller ones.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">JIRA: <a href="https://mercadolibre.atlassian.net/browse/JIRA-123">JIRA-123</a></td>
    </tr>
  </tbody>
</table>

## Current coverage for MercadoPagoSDKV4.framework is `2.89%`
✅ *No files affecting coverage found*

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_danger_pr">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by pedro-nadolny** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a <a href="https://github.com/mercadopago/px-ios/blob/develop/CHANGELOG.md">CHANGELOG.md</a> entry. The changelog entry should be equal to the PR title in the correct section: <strong>Added danger and swiftlint to the project</strong></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR does not include tests, review carefully</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The <code>Podfile</code> was updated</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR, consider splitting into smaller ones.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">JIRA: <a href="https://mercadolibre.atlassian.net/browse/JIRA-123">JIRA-123</a></td>
    </tr>
  </tbody>
</table>

## Current coverage for MercadoPagoSDKV4.framework is `2.89%`
✅ *No files affecting coverage found*

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_danger_pr">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Pull request merged**